.github/workflows/ci.yml

```
@@ -76,6 +76,8 @@ jobs:
           noPrefix: true
           awsAccessKeyId: ${{ secrets.AWS_ACCESS_KEY_ID }}
           awsSecretAccessKey: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          deployRunUrl: https://portal-report.concord.org/__deployPath__/
```

It'd probably be better to use this URL: 
```suggestion
          deployRunUrl: https://portal-report.concord.org/__deployPath__/?portal-dashboard
```
That way it will bring up the more commonly used dashboard instead of original report.